recipe/meta.yaml

```diff
@@ -141,7 +141,7 @@ outputs:
         - {{ pin_subpackage('hyperspy-base', exact=True) }}
```

Comment:
I think we should remove this line in order to make installation of development version more easy, by doing:
```
conda install hyperspy-dev hyperspy --only-deps
pip install -e .
```
At the moment, we need to install `hyperspy` to get all the dependency install and then uninstall it (but keeping the dependencies)